api/lib/reducers/cosmosV0-reducers.js

```diff
@@ -303,6 +308,42 @@ function validatorReducer(networkId, signedBlocksWindow, validator) {
   }
 }
 
+function validatorProfileReducer(
+  validator,
+  primitiveValidator,
+  validatorProfile,
+  totalStakedAssets,
+  numberStakers,
+  network,
+  feed
+) {
+  return {
+    ...validator,
+    profile: {
+      name: primitiveValidator.description.moniker,
```

Comment:
what is `primitiveValidator`?

Reply:
from the chain. I can rename it to `chainValidator`

Reply:
isn't our convention normally something like:

```
validator
enrichedValidator
```

Reply:
I didn't know we had a convention about this :shrug: I think it hasn't been that well respected throughout our repo

Reply:
maybe. let's use it though. i think it is a good convention.

Reply:
OK, let me do this

Reply:
Done